ticdc/ticdc-overview.md

```diff
@@ -67,6 +67,21 @@ The components in the preceding architecture diagram are described as follows:
 
 As shown in the preceding architecture diagram, TiCDC supports replicating data to TiDB, MySQL, and Kafka databases.
 
+## Best practices
+
+- When using TiCDC to replicate data between two TiDB clusters and the network latency between them is greater than 100 ms, it is recommended that you deploy TiCDC in the region (IDC) where the downstream TiDB cluster is located.
+- TiCDC only replicates the table that has at least one **valid index**. A **valid index** is defined as follows:
+
+    - A primary key (`PRIMARY KEY`) is a valid index.
+    - A unique index (`UNIQUE INDEX`) is valid if every column of it is explicitly defined as non-nullable (`NOT NULL`) and the index does not have the virtual generated column (`VIRTUAL GENERATED COLUMNS`).
+
+- To use TiCDC in DR scenarios, you need to configure [redo log](/ticdc/ticdc-sink-to-mysql.md#eventually-consistent-replication-in-disaster-scenarios).
+- When replicating a wide table with a large single row (greater than 1K), it is recommended that you configure [per-table-memory-quota](/ticdc/ticdc-server-config.md) so that `per-table-memory-quota` = `ticdcTotalMemory`/(`tableCount` * 2). `ticdcTotalMemory` is the memory of a TiCDC node, and `tableCount` is the number of target tables that a TiCDC node replicates.
```

Comment:
```suggestion
- When you replicate a wide table with a large single row (greater than 1K), it is recommended that you configure [`per-table-memory-quota`](/ticdc/ticdc-server-config.md) so that `per-table-memory-quota` = `ticdcTotalMemory`/(`tableCount` * 2). `ticdcTotalMemory` is the memory of a TiCDC node, and `tableCount` is the number of target tables that a TiCDC node replicates.
```